pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "cmake", "ninja"]
+build-backend = "setuptools.build_meta"
```

Comment:
So we're doing this now?

Reply:
This is the way I got things to be 'reliable'; I hate that I don't know the full interactions of this; however, it seems that it doesn't work as well as I expected - I will go back to the drawing board.